th/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,175 @@
+---
+version: 1.0.0
+title: การทำงานพร้อมกัน (Concurrency)
+---
+
+หนึ่งในจุดขายหลักของ Elixir คือการรองรับการทำงานพร้อมๆ กัน ซึ่งเป็นผลพลอยได้จากการนำ Erlang VM (BEAM) มาใช้งาน ส่งผลให้การจัดการการทำงานพร้อมกันใน Elixir นั้นง่ายกว่าที่คิดมากๆ
+
+โดยรูปแบบการทำงานพร้อมๆ กันนั้น มีความจำเป็นต้องพึ่งพา Actors​ ซึ่งเป็นโปรเซสที่สามารถสื่อสารกับโปรเซสอื่นๆ ได้โดยผ่านการส่งข้อความหากัน
```

Comment:
Actors -> actor

Let use the singular term for the entire translation according to @iboss-ptk' s reason here: https://github.com/elixirschool/elixirschool/pull/1194#discussion_r143441228, which I also agree with him. :)

Sounds good?

Reply:
Sounds great! Thinking about it, singular term feels a lot more natural.